src/features/chat/common.ts

```diff
@@ -0,0 +1,6 @@
+export interface ChatUser {
```

Comment:
Please use type instead of interface and add T as well

Reply:
changed